instrumentation/pg/opentelemetry-instrumentation-pg.gemspec

```diff
@@ -26,7 +26,16 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 3.0'
 
   spec.add_dependency 'opentelemetry-api', '~> 1.0'
-  spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.22.1'
+  # TODO: Return base version constraint. When constraint present, following error is raised:
```

Comment:
This TODO should probably be resolved along with those in the trilogy and mysql2 instrumentations, seems bad to drop the version constraint. I'm not sure I understand the bundler error though?

Reply:
I totally agree with you. I don't want to drop the version constraint and won't consider this done until I can find a fix for this issue. My hope was that this had been seen before in the repo and someone would know the right fix, but it sounds like that might not be the case! I might need to go back to the drawing board on how to create the new apps to see if I can stop it from happening.

Reply:
can surface this with the broader SIG, seems like i'm probably just missing something trivial.

Reply:
Sounds good, thank you. I saw this after I posted about it in the #otel-ruby channel. Here's the Slack link: https://cloud-native.slack.com/archives/C01NWKKMKMY/p1688672784020679

Reply:
The resolution was to add Gemfile entries in a `:test` block for the new helpers.